frappe/email/doctype/email_account/email_account.js

```diff
@@ -197,13 +203,22 @@ frappe.ui.form.on("Email Account", {
 	},
 
 	show_oauth_authorization_message(frm) {
-		if (frm.doc.auth_method === "OAuth" && !frm.doc.refresh_token) {
-			let msg = __(
-				'OAuth has been enabled but not authorised. Please use "Authorise API Access" button to do the same.'
-			);
-			frm.dashboard.clear_headline();
-			frm.dashboard.set_headline_alert(msg, "yellow");
-		}
+		frappe.call({
+			method: "frappe.integrations.doctype.connected_app.connected_app.check_active_token",
```

Comment:
This should only happen if the auth method is OAuth. I'm seeing errors on basic-auth accounts as well.